app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -330,6 +336,12 @@ def number_of_true_checkboxes(checkbox_array_for_line)
         checkbox_array_for_line.sum { |a| a == true ? 1 : 0 }
       end
 
+      def interest_on_gov_bonds
+        interest_reports = @intake.direct_file_json_data.interest_reports
+        interests_on_gov_bonds = interest_reports&.map(&:interest_on_government_bonds)
+        interests_on_gov_bonds.sum.round
```

Comment:
[pebble] `interests_on_gov_bonds` could be `nil` which would run into issues with `.sum.round` 

should use `interests_on_gov_bonds&.sum&.round` or some other kind of safety navigation here

Reply:
Done, ty for catching!